src/scroll-timeline-css-parser.js

```diff
@@ -425,7 +429,21 @@ export class StyleParser {
   }
 
   extractTimelineName(shorthand) {
-    return this.findMatchingEntryInContainer(shorthand, this.scrollTimelineOptions);
+    let timelineName = null;
+    let toBeReplaced = null; // either timelineName or anonymousTimeline
+
+    const anonymousStart = shorthand.indexOf("scroll(");
```

Comment:
Using the regex /scroll\(([^\)]*)\)/  is likely more rigorous. Note the capture group for the scroll arguments.